Makefile

```diff
@@ -99,3 +133,57 @@ verify-shellscripts-initramfs:
 		--severity error \
 		defaults/linuxrc \
 		defaults/initrd.scripts
+
+$(BUILD_DIR)/build-config:
+# $(addprefix $(BUILD_DIR)/temp/,$(TEMPFILES))
+	install -d $(BUILD_DIR)
+	echo ${PREFIX} > $(BUILD_DIR)/PREFIX
+	echo ${BINDIR} > $(BUILD_DIR)/BINDIR
+	echo ${SYSCONFDIR} > $(BUILD_DIR)/SYSCONFDIR
+	echo ${MANDIR} > $(BUILD_DIR)/MANDIR
+	touch $(BUILD_DIR)/build-config
+
+$(BUILD_DIR)/software.sh:
+	install -d $(BUILD_DIR)/temp/
+	echo -e $(SOFTWARE_VERSION) > $(BUILD_DIR)/temp/versions
+	cat $(BUILD_DIR)/temp/versions defaults/software.sh > $(BUILD_DIR)/software.sh
+
+$(BUILD_DIR)/doc/genkernel.8.txt:
+	install -d $(BUILD_DIR)/doc/
+	cp doc/genkernel.8.txt $(BUILD_DIR)/doc/genkernel.8.txt
```

Comment:
you seem to mix `cp` and `install` a lot; this can be probably be make more consistent.

Reply:
addressed in commit [e87c328](https://github.com/gentoo/genkernel/commit/e87c328eac10865f4976cf2f96d3a5f86bbfe88c)